tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "moduleResolution": "node",
-    "target": "es5",
+    "target": "es6",
```

Comment:
Why this change? Could we replace with https://github.com/tsconfig/bases instead of creating a custom tsconfig?

In any case, this should trigger a release, right?

Reply:
I found an issue during build time that according to some issues was fixed by that. I just rolled back to `es5` and don't see it anymore happening. Maybe it was an issue of not using the same TS version in all packages simultaneously. So you're suggesting installing and adding `"extends": "@tsconfig/recommended/tsconfig.json"` to the TSconfig?

Reply:
Yep. Of course only if it doesn't require any more work or changes in the code.